### PR TITLE
Navbar option IsSpaced 

### LINKED ISSUE #180: Navbar: is-spaced
Navbar.IsSpaced is missing, which adds extra padding to a navbar.

It is something that works and I can use it, it is in the source files, Bulma docs uses it, but it is not reported in the documentation...

https://github.com/jgthms/bulma/blob/master/sass/components/navbar.sass#L278

image

### PR DESCRIPTION
**Pull request by selketjah**
Fixes #180

### CODE REVIEW

File: src/Fulma/Components/Navbar.fs [hunk 1]
```diff
@@ -19,6 +19,9 @@ module Navbar =
         /// Add `is-fixed-bottom` class
         /// You also need to add `has-navbar-fixed-bottom` to your html tag
         | [<CompiledName("is-fixed-bottom")>] IsFixedBottom
+        /// Add `is-spaced` class
+        /// Sets Top and Bottom paddings with 1 rem, Left and Right paddings with 2 rem
```
Comment (MangelMaxime) [inline src/Fulma/Components/Navbar.fs]: I don't think we can add space info in the documentation.

The `1rem` and `2rem` can be changed using variables.

Comment (selketjah) [reply src/Fulma/Components/Navbar.fs]: "Increases horizontal and vertical padding of the navbar" would be a better fit then.

Comment (selketjah) [reply src/Fulma/Components/Navbar.fs]: done

**Comment by MangelMaxime** [comment]
Thank you

**Comment by MangelMaxime** [comment]
Released in version 2.0.0

**Pull request merged**